fetch-utils.js

```diff
@@ -0,0 +1,48 @@
+const SUPABASE_URL = 'https://nxkfcrgevcuksvdnaugc.supabase.co';
+const SUPABASE_KEY = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Im54a2ZjcmdldmN1a3N2ZG5hdWdjIiwicm9sZSI6ImFub24iLCJpYXQiOjE2NDQzNDE0NjAsImV4cCI6MTk1OTkxNzQ2MH0.9e3rnFUxMChSkwzJQcBImTDSwY_4VDtfmLDUZ-dY7gc';
+
+const client = supabase.createClient(SUPABASE_URL, SUPABASE_KEY);
+
+function checkError({ data, error }) {
+    return error ? console.error(error) : data;
```

Comment:
Here I would hit the lightbulb and hit eslint disable console.log that will stop the warnings